packages-ts/starknet/test/fundAccount.test.ts

```diff
@@ -0,0 +1,80 @@
+import { assert } from 'chai'
+import { account } from '@chainlink/starknet'
+import { Account, ec, SequencerProvider, stark } from 'starknet'
+import { DEVNET_URL, ERC20_ADDRESS_DEVNET, ERC20_ADDRESS_TESTNET } from '../src/utils'
+
+describe('fundAccount', function () {
+  this.timeout(900_000)
+  let alice: Account
+  let bob: Account
+  let provider: SequencerProvider
+  let erc20_address: string
+
+  const opts = account.makeFunderOptsFromEnv()
+  const funder = account.Funder(opts)
+
+  before(async function () {
+    const gateway_url = process.env.NODE_URL || DEVNET_URL
+    provider = new SequencerProvider({ baseUrl: gateway_url })
+
+    if (gateway_url === DEVNET_URL) {
+      erc20_address = ERC20_ADDRESS_DEVNET
+    } else {
+      erc20_address = ERC20_ADDRESS_TESTNET
+    }
+
+    const alice_starkKeyPair = ec.genKeyPair()
```

Comment:
Please use camelCase instead of snake_case

```suggestion
    const aliceStarkKeyPair = ec.genKeyPair()
```